spec/models/like_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+
+feature 'Liking photos' do
+
+  context 'Signed in' do
+
+    let!(:sea) {Photo.create(title:'Awesome sea',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space missing after colon.